app/graphql/connections/hosts_connection.rb

```diff
@@ -0,0 +1,6 @@
+Connections::HostsConnection = Types::Host.define_connection do
```

Comment:
In docs I see now another way of defining connection: https://github.com/rmosolgo/graphql-ruby/blob/master/guides/relay/connections.md#connection-types

As I understand from docs, I think we could DRY code here a little with:


```
class ConnectionWithTotalCountType < GraphQL::Types::Relay::BaseConnection
  field :total_count, Integer, null: false
  def total_count
    object.nodes.size
  end
end
```

and then for each type we should define `EdgeType`:

```
class HostEdgeType < GraphQL::Types::Relay::BaseEdge
  node_type(HostType)
end
```

and in the types:

```
field :hosts, ConnectionWithTotalCountType, connection: true, function: Queries::PluralField.new(type: Types::Host, model_class: ::Host::Managed)
```

I didn't test it by myself, but I think it could be possible. WDYT?